src/runtime/multiplexer-dom-particle.ts

```diff
@@ -28,19 +28,24 @@ export class MultiplexerDomParticle extends TransformationDomParticle {
       particleHandleName: string,
       hostedParticle: ParticleSpec,
       handles: ReadonlyMap<string, Handle>,
-      arc) {
+      arc: InnerArcHandle) {
     const otherMappedHandles: string[] = [];
     const otherConnections: string[] = [];
     let index = 2;
     const skipConnectionNames = [listHandleName, particleHandleName];
+
     for (const [connectionName, otherHandle] of handles) {
       if (skipConnectionNames.includes(connectionName)) {
         continue;
       }
       // TODO(wkorman): For items with embedded recipes we may need a map
       // (perhaps id to index) to make sure we don't map a handle into the inner
       // arc multiple times unnecessarily.
-      otherMappedHandles.push(`use '${await arc.mapHandle(otherHandle.storage)}' as v${index}`);
+
```

Comment:
There's some oddities where the types specified don't seem to match with the calling convention...  Also see above where there's a distinction between the Recipe version of Handle and the base Handle class.

fyi to @shaper who might have historical knowledge...